include/glow/Backends/BackendUtils.h

```diff
@@ -24,7 +24,13 @@ namespace runtime {
 
 /// An enum to indicate what type each symbol in the bundle is. Current options
 /// are activations, constants, and placeholders.
```

Comment:
Comment here is stale now -- you could probably just remove the second sentence entirely.